srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -113,6 +124,10 @@ public boolean isExpressionEnabled()
         return fExpressionEnabled;
     }
     
+    /**
+     * Sets if this expression is to be evaluated.  True means the expression is evaluated.
+     * False means the expression is not evaluated.
```

Comment:
Haha, well you did clarify but I was thinking more, "If the expression is not evaluated, `evaluate()` always returns true."